docker-compose.yml

```diff
@@ -108,6 +108,19 @@ services:
     networks:
       - primary
 
+  dbmate:
```

Comment:
Instead of using another container, I'd prepare a docker entry point that runs the migration optionally before the controlplane start.